### PR TITLE
[docs] APM Elasticsearch output config

### PR DESCRIPTION
**Pull request by bmorelli25**
This PR updates the ES output config documentation to include examples pertinent to APM.

@dedemorton - hoping to get some advice on this one. This is kind of an ugly solution that introduces a bunch of new `ifndef::apm-server[]`/`ifdef::apm-server[]` conditionals. I played around with other solutions based off attributes, but I couldn't find anything better 🤷‍♂. 

Related to https://github.com/elastic/apm-server/issues/2130

### CODE REVIEW

File: libbeat/docs/outputconfig.asciidoc [hunk 1]
```diff
@@ -79,7 +79,7 @@ output.elasticsearch:
 
 If the Elasticsearch nodes are defined by `IP:PORT`, then add `protocol: https` to the yaml file.
 
-[source,yaml]
+["source","yaml",subs="attributes,callouts"]
```
Comment (bmorelli25) [inline libbeat/docs/outputconfig.asciidoc]: FYI: This line is unrelated to this PR, looks like a bug:
Screen Shot 2019-05-02 at 12 05 08 PM

Comment (dedemorton) [inline libbeat/docs/outputconfig.asciidoc]: Good catch! You can probably just use `subs="attributes"` since this example doesn't have callouts. Doesn't hurt either way, tho....

Comment (bmorelli25) [reply libbeat/docs/outputconfig.asciidoc]: Oops. You're right, not sure why I added callouts in there 🐠. I guess I'll leave it since it doesn't hurt (but mostly so I don't have to wait for 5 successful tests again 😄)
